src/routes/Authentication.js

```diff
@@ -1,10 +1,21 @@
 import express from 'express';
 import { createAgent, loginAgent } from '../controllers/AgentAuthentication';
-import { validateUserInput } from '../middleware';
+import {
+  agentProperties, allProperties, createProperty,
+  deleteProperty, editProperty, getPropertyById
+} from '../controllers/propertyInfo/propertyController';
+import { validatePropertyInput, validateUserInput } from '../middleware/middleware';
+import { verifyToken } from '../middleware/auth';
 
 const indexRouter = express.Router();
 
 indexRouter.post('/signup', validateUserInput, createAgent);
 indexRouter.post('/login', loginAgent);
+indexRouter.get('/properties', allProperties);
+indexRouter.get('/property/:id', getPropertyById);
+indexRouter.post('/post-property', verifyToken, validatePropertyInput, createProperty);
```

Comment:
You can't put agent property routes inside of the `Authentication` route, the right thing to do will be to put them inside a new file named `PropertyRoutes.js`

Reply:
Also, your route can't be named `post-property` the HTTP method name already says `POST` you shouldn't be using post in the route name again, it's better named `/property`